src/Jira/Api/Client/CurlClient.php

```diff
@@ -52,10 +53,10 @@ public function __construct()
      */
     public function sendRequest($method, $url, $data = array(), $endpoint, AuthenticationInterface $credential, $isFile = false, $debug = false)
     {
-        if (!($credential instanceof Basic)) {
-            throw new \Exception(sprintf("CurlClient does not support %s authentication.", get_class($credential)));
+        if (!($credential instanceof Basic) && !($credential instanceof Anonymous)) {
+            throw new \Exception(sprintf("PHPClient does not support %s authentication.", get_class($credential)));
```

Comment:
I haven't noticed, that exception message was containing class name. Please change `PHPClient` back to `CurlClient`.